src/components/navBar/navBar.css

```diff
@@ -0,0 +1,96 @@
+.navBar-container {
+    display: flex;
+    flex-direction: row;
+    width: 1440px;
+    height: 53px;
+    padding: 14px 123px 5px;
+    background-color: var(--white);
+}
+
+.logo-icon {
+    width: 33px;
+    height: 33px;
+    margin-left: 123px;
+    margin: 0 15px 1px 0;
+    object-fit: contain;
+}
+
+.logo{
+    width: 115px;
+    height: 32px;
+    margin: 1px 31px 1px 15px;
+    font-family: HelveticaNeue;
+    font-size: 27px;
+    font-weight: 100;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 1.35px;
```

Comment:
Same as the font-size comment.

Reply:
This comment is relevant to all the replies above:
The problem is that I was relying on the built in css in the psd, which kind of ruined the whole css and the work process.
I learned my lesson after hours of unnecessary  debugging :)
Today I'll refresh my memory about pixel perfect, and tomorrow will refactor the code.

Reply:
Yeah there are a few caveats when using design tools that give out css, can be a bit misleading.
Good job so far!